ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -8,22 +8,198 @@ import StatusQ.Core.Theme 0.1
 import SortFilterProxyModel 0.2
 import AppLayouts.Profile.stores 1.0 as ProfileStores
 import AppLayouts.Wallet.stores 1.0 as WalletStore
+import AppLayouts.stores.Messaging 1.0 as MessagingStores
 
+// WIP: Previous reorganization step before refactoring `RootStore`
```

Comment:
Still WIP?

Reply:
Yes yes, as said, I added lots of comments to have a clear guide of how to proceed in the next prs!